tests/v2/validation/deleting/delete_init_machine_test.go

```diff
@@ -48,21 +48,21 @@ func (d *DeleteInitMachineTestSuite) TestDeleteInitMachine() {
 	err = v1.ConvertToK8sType(cluster, &updatedCluster)
 	require.NoError(d.T(), err)
 
-	name := "K3S"
+	var name string
 
 	if strings.Contains(updatedCluster.Spec.KubernetesVersion, "rke2") {
 		name = "RKE2"
+	} else {
+		name = "K3S"
 	}
-	require.NotContains(d.T(), updatedCluster.Spec.KubernetesVersion, "-rancher")
```

Comment:
does this run on rke1 now?

Reply:
It doesn't, was this intentional? I was just generally cleaning, I can put it back though.

Reply:
Resolved.